web/src/components/timeslider_new.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useMemo, useState } from 'react';
+import { scaleTime } from 'd3-scale';
+import sortedIndex from 'lodash.sortedindex';
+
+import TimeAxis from './graph/timeaxis';
+import { useRefWidthHeightObserver } from '../hooks/viewport';
+import TimeSliderTooltip from './tooltips/timeslidertooltip';
+import TimeControls from './timeControls';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import thumbImg from '../../public/images/slider-thumb.svg';
+
+const AXIS_HORIZONTAL_MARGINS = 12;
+
+const StyledInput = styled.input`
+  -webkit-appearance: none;
+  width: 100%;
+  margin-top: 5px;
+  height: 26px;
+  background: transparent;
+  padding: 0;
+  overflow: visible;
+  border: none;
+  margin: 5px 0;
+
+  &:focus {
+    outline: none;
+  }
+
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    height: 26px;
+    width: 26px;
+    border-radius: 17px;
+    background: #ffffff;
+    cursor: pointer;
+    margin-top: -8px;
+    transition: box-shadow 0.4s;
+    box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.15);
+    border: 1px solid lightgray;
+    background-position: center center;
+    background-size: 12px;
+    background-repeat: no-repeat;
+    background-image: url(${thumbImg});
```

Comment:
Does it fail if we import them like we do in the stylesheets?
<img width="614" alt="Screenshot 2022-05-10 at 12 11 12" src="https://user-images.githubusercontent.com/3296643/167605600-ade3e860-6098-4e19-a07f-8466d6079deb.png">

If yes, then @VIKTORVAV99's suggestion is probably best - I guess it will in future be nice to able to use SVGs anyway :)

Reply:
No it won't (I think), the difference here is that webpack is trying to import the contents of the SVG file (and inline it) and use that while in the stylesheet we just do "use this file" basically.

Reply:
Just in the spirit of finalizing this PR I've used the .scss file. We will have to look at this very soon again, as we'll need to remove the entire file.

Then we can handle the SVG loader in a separate PR

Reply:
The scss files are getting removed? 👀

Last I checked the postcss don't support styled components so we would lose autoprefixer and quite a few optimizations doing that?